sdk/formrecognizer/azure-ai-formrecognizer/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "python",
+  "TagPrefix": "python/formrecognizer/azure-ai-formrecognizer",
+  "Tag": "python/formrecognizer/azure-ai-formrecognizer_153a841398"
```

Comment:
This was all automatically generated?

Reply:
Yep! The only thing that will change in the future is the `Tag` field when new recordings are pushed -- it's just a pointer to the tag in the assets repo that contains the recordings you need 🙂